### PR TITLE
Storybook Updates

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Use latest themes in Storybook
- Fix some storybook stories.

### CODE REVIEW

**Comment by emclaug2** [comment]
For some unknown reason, the drawer nav rail story isn't fixed in dev.  This'll require a small fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v4.2.1 (Not Published Yet)
```
Comment (joebochill) [inline CHANGELOG.md:3]: Since this is a new feature, I think this should be a minor release 4.3.0, but we can update that when we're closer to publishing

**Review by joebochill — APPROVED** [review]

**Comment by emclaug2** [comment]
I think it'd be a good idea to publish the storybook changes in its current state since the Drawer header has been incorrect for a week now.   I'm opening this PR for review.

**Review by jeffvg — APPROVED** [review]
looks good to me

**Pull request merged**